Feature/print format

shantanuthatte opened the pull request:
Implemented formatting of display time by providing a `format` option. Formats are:
`%h` - Hours without leading zero.
`%m` - Minutes without leading zero.
`%s` - Seconds without leading zero.
`%H` - Hours with leading zero.
`%M` - Minutes with leading zero.
`%S` - Seconds with leading zero.

**NOTE:** If you pass `{seconds: 69, format: '%m:%s'}` you would still get `1:09` as output. If `1:9` is required, it's a simple fix by adding `parseInt()` to `formatDef` for `%h`, `%m` and `%s`.

walmik commented:
Thank you very much for implementing this much desired functionality in such a systematic manner! Please bump version in package.json to `0.3.0` and I ll merge this.

shantanuthatte commented:
I've updated `package.json` and edited `README.md`. Updated `README.md` is (here)[https://github.com/shantanuthatte/timer.jquery/blob/feature/print-format/README.md].

walmik commented:
Excellent!
